healthcheck/contrib/django/status_endpoint/tests/test_views.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import connections
+from django.test import TestCase, override_settings
+from django.test.client import Client
+
+from healthcheck.contrib.django.status_endpoint import views
+
+
+if not settings.configured:
+    settings.configure(
+        DATABASE_ENGINE='sqlite3',
+        DATABASES={
+            'default': {
+                'NAME': ':memory:',
+                'ENGINE': 'django.db.backends.sqlite3',
+                'TEST_NAME': ':memory:',
+            },
+        },
+        DATABASE_NAME=':memory:',
+        TEST_DATABASE_NAME=':memory:',
+        INSTALLED_APPS=['healthcheck.contrib.django.status_endpoint'],
+        ROOT_URLCONF='',
+        DEBUG=False,
+        SITE_ID=1,
+        TEMPLATE_DEBUG=True,
+        PROJECT_ROOT=os.path.dirname(os.path.abspath(__file__)),
+        ALLOWED_HOSTS=['*'],
+    )
+
+
+class StatusEndpointViewsTestCase(TestCase):
+    urls = 'healthcheck.contrib.django.status_endpoint.urls'
+
+    @override_settings(
+        STATUS_CHECK_DBS=True,
+        STATUS_CHECK_FILES=('/etc/quiesce',)
+    )
+    def test_default_checks(self):
+        response = Client().get(reverse(views.status))
```

Comment:
You don't need to change this, but I wanted to point out something about the Django test client that you're using here for future reference:

It really, really, really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really,  really, really sucks. These aren't actually unit tests because every test is hitting every part of the django stack from middlewares to routing. They're actually integration tests, but they're bad integration tests because they are hitting some monkey patched version of the entire Django stack instead of the real deal.

It is much better to use RequestFactory to test views, which actually lets you hit them in isolation. If you care about verifying routing, you can also do that in isolation using a resolver. Here's an example: https://github.com/thomasw/djproxy/blob/master/tests/url_generate_routes_tests.py#L9-L35